booting_windows/setup.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euo pipefail
+
+# Path to a working Windows 2019 image:
+WINDOWS_DISK=$(WORKSPACE)/win2019.raw
+
+START_PATH=$(realpath .)
+
+### Downloading and extracting ovmf EFI firmawre ###
+mkdir -p "${EFI_WORKSPACE}"/downloads
+cd "${EFI_WORKSPACE}"/downloads
+
+FIRMWARE_URL=https://www.kraxel.org/repos/jenkins/edk2/edk2.git-ovmf-x64-0-20190516.1084.g3604174718.noarch.rpm
```

Comment:
Link no longer works. Here's a newer one:
https://www.kraxel.org/repos/jenkins/edk2/edk2.git-ovmf-x64-0-20190704.1194.g073f2cede8.noarch.rpm

No idea if it's compatible, etc. :)

Reply:
Replaced. See b28937d6